src/main/java/cz/brmlab/yodaqa/pipeline/structured/StructuredPrimarySearch.java

```diff
@@ -179,6 +196,19 @@ protected void propertyToAnswer(JCas jcas, PropertyValue property,
 
 		ai.addToIndexes();
 	}
+	protected int getSourceID(String url, String label, JCas questionView){
+		if (sourceIDs.containsKey(url)) {
+			return sourceIDs.get(url);
+		}
+
+		int sourceID = SourceIDGenerator.getInstance().generateID();
+		AnswerSourceStructured asf = new AnswerSourceStructured("structured", url, label);
```

Comment:
Let's use a string constant in AnswerSourceStructured for the "structured" literal.

The reasoning is that these are machine-readable identifiers, not meant necessarily directly for the end-user, they are part of our API.  Therefore, all possible values should be at a single place to make the code self-documenting.